jenkins/catapult/environment.py

```diff
@@ -28,6 +28,18 @@ def system_bin_path(self):
 
 		raise RuntimeError('unable to detect system bin path')
 
+	@property
+	def local_lib_path(self):
```

Comment:
why did we need to add this ?

Reply:
Due to openssl getting copied to all the test base images.  Fedora also now ships with openssl 3.
Local lib path is different for Fedora vs Ubuntu.